frontend/src/pages/audio/_id/index.vue

```diff
@@ -1,32 +1,31 @@
 <template>
-  <main :id="skipToContentTargetId" tabindex="-1">
-    <div v-if="backToSearchPath" class="w-full px-4 pb-4 md:px-8">
-      <VBackToSearchResultsLink
-        :id="$route.params.id"
-        :href="backToSearchPath"
-      />
-    </div>
-
-    <template v-if="audio">
-      <VAudioTrack
-        layout="full"
-        :audio="audio"
-        class="main-track"
-        @interacted="sendAudioEvent($event, 'AudioDetailPage')"
-      />
-      <div
-        class="mx-auto mt-10 flex flex-col gap-10 px-6 lg:mt-16 lg:max-w-5xl lg:gap-16"
-      >
-        <VMediaReuse data-testid="audio-attribution" :media="audio" />
-        <VAudioDetails data-testid="audio-info" :audio="audio" />
-        <VRelatedAudio @interacted="sendAudioEvent($event, 'VRelatedAudio')" />
-      </div>
+  <main :id="skipToContentTargetId" tabindex="-1" class="relative flex-grow">
+    <VSafetyWall v-if="isHidden" :media="audio" @reveal="reveal" />
+    <template v-else>
```

Comment:
```suggestion
    <template v-else-if="audio">
```
Would remove the need to add `v-if="audio"` to the nested items.